src/components/NewGame/GameRules/GameList.ts

```diff
@@ -0,0 +1,174 @@
+import { GameListItem } from '../NewGameTypes';
+
+export const gamesList: GameListItem[] = [
+  {
```

Comment:
Wouldn't it be easier to use enums for game properties. In case some property changes functionality/default value etc., you will end up having to do quite a tedious rework, if find + replace can be considered tedious.